ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -710,6 +710,27 @@ func (ipfs *Connector) BlockGet(ctx context.Context, c cid.Cid) ([]byte, error)
 	return ipfs.postCtx(ctx, url, "", nil)
 }
 
+// // FetchRefs asks IPFS to download blocks recursively to the given depth.
```

Comment:
Why is this commented out?

Reply:
not used. 60% sure that we will use it one day for something...